atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransaction.java

```diff
@@ -660,6 +669,12 @@ private void verifyColumnRanges(Transaction readOnlyTransaction) {
         }
     }
 
+    private Meter getTransactionConflictsMeter() {
+        // TODO(hsaraogi): add table names as a tag
+        return metricRegistry.meter(
+                MetricRegistry.name(SerializableTransaction.class, "SerializableTransactionConflict"));
+    }
+
```

Comment:
Do we have any blockers to factoring out both `getTransactionConflictsMeter().mark()` and the subsequent `throw TSCE.create(...)` to its own method? I think that would be cleaner (if it's possible, of course).